### ISSUE TITLE
Photon: do not apply Photon when doing an AJAX request in the admin

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#722]
We already exclude Photon from being applied on admin, but not when working with AJAX in the admin.

Reported here:
- http://wordpress.org/support/topic/jetpack-photon-with-admin-ajaxphp?replies=1

Original trac ticket:
- https://plugins.trac.wordpress.org/ticket/1885

**Comment by alexisbellido** [comment]
I just had an encounter with this issue. Any updates?

**Comment by kraftbj** [comment]
Not at this time. It's marked in our Community milestone, which means at this time we're open to PRs but do not actively have this issue on our roadmap.

**Comment by kraftbj** [comment]
To confirm the issue: You do want Photon to change URLs for AJAX requests and right now, it isn't since all ajax requests return true for `is_admin`. Is that correct or did I misunderstand?

**Comment by alexisbellido** [comment]
Yes, that's my case. In fact, I'm making these ajax calls from public facing pages, not the admin, but I think the root of the problem is the same described here: the check for is_admin() in https://github.com/Automattic/jetpack/blob/master/class.photon.php#L421

**Comment by kraftbj** [comment]
👍 Thanks for the clarification. Within WP, all AJAX calls will return true for `is_admin` no matter the origin (odd, I know), so that line is most likely the offending case.
